TTHAnalysis/python/plotter/susy-ewkino/2lss/includes/mca_prompt.txt

```diff
@@ -0,0 +1,40 @@
+########### prompt_WZ : WZTo3LNu : xsec*1.02 : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kOrange, Label="WZ", NormSystematic=0.25
+########### Already included!! It's the first to keep it on top in the plots.
+
+######### ZZH
+################################################
+prompt_ZZH         : ZZTo4L_part1 + ZZTo4L_part2   : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kGreen+1,  Label="ZZ/H",   NormSystematic=0.25
+prompt_ZZH         : VHToNonbb  : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kGreen+1, Label="ZZ/H", NormSystematic=0.25
+prompt_ZZH         : GGHZZ4L  : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kGreen+1, Label="ZZ/H", NormSystematic=0.25
+
+######### ttX
+################################################
+rares_ttW   : TTWToLNu_ext1 + TTWToLNu_ext2_part1 + TTWToLNu_ext2_part2 : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttZ   : TTZToLLNuNu                 : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttZ   : TTHnobb_pow        : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : TTLLJets_m1to10             : 0.0493 : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : TTJets_DiLepton_part1 + TTJets_DiLepton_part2 : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : TTJets_SingleLeptonFromT_part1 +  TTJets_SingleLeptonFromT_part2 + TTJets_SingleLeptonFromT_part3 : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : TTJets_SingleLeptonFromTbar : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : TTTT        : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : tZq_ll      : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : tZW_ll      : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : TTWToQQ : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
+rares_ttX   : TTZToQQ : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-3, Label="ttX",   NormSystematic=0.20
```

Comment:
should we really use those two? I expect to get very low yield esp. due to the prompt matching. The "proper" TTV samples are already included above.